### DISCUSSION TITLE
Unexpected Generator 'TextColorToGenerator' warnings when any build error occurs.

### DISCUSSION THREAD

**Discussion by DanTravison** [CommunityToolkit/Maui#1417] [Q&A]
I have two solutions that contain a mix of Maui and pure .Net 7 projects.  In both cases one of the projects references CommunityToolkit.Maui 5.3.0.

Now, whenever I have a build error in any of the projects, I also get the following warning:

CSC : warning CS8785: Generator 'TextColorToGenerator' failed to generate source. It will not contribute to the output and compilation errors may occur as a result. Exception was of type 'Exception' with message 'There's no .NET MAUI referenced in the project.'

The curious part is it occurs on project chains that have no Maui dependencies, so I don't understand why it's even being run during the build for those projects.

I've tried cleaning the solution via Build/Clean Solution as well as manually cleaning all generated files and the .vs directory with no change in behavior.

This started occurring recently (last few days) and in that time I updated Visual Studio Community to Version 17.8.0 Preview 2.0. 
I don't recall if it was occurring before the update or only after the update.

Is this a Visual Studio issue or is there something else I'm missing?
Thank.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
